docker/msmtprc

```diff
@@ -0,0 +1,11 @@
+defaults
+auth           off
+tls            off
+tls_starttls   off
+
+account        mailsink
+host           mailsink
+port           1025
+from           accounts@wmflabs.org
```

Comment:
Question: what's the reason for this email address to be hardcoded here?

Reply:
msmtp's "sendmail" mode requires this setting, which actually sets the "envelope from" used when negotiating with the SMTP server. It's independent of and shouldn't affect the `From` header in the email. Since we're only talking to our mail sink, this can actually be any value, I just figured this would be a reasonable one.